yellowbrick/features/missing.py

```diff
@@ -0,0 +1,129 @@
+# yellowbrick.features.missing
+# Feature importance visualizer
+#
+# Author:  Nathan Danielsen <nathan.danielsen@gmail.com>
+# Created: Fri Mar 24 8:17:36 2018 -0500
+#
+# Copyright (C) 2018 District Data Labs
+# For license information, see LICENSE.txt
+#
+# ID: missing.py [] nathan.danielsen@gmail.com.com $
+
+"""
+Implementation of missing values visualizers
+
+To Include:
+- Bar
+- Density Matrix (by time, specifiable index)
```

Comment:
I'm super excited about this! 

The density matrix =seems similar to [Lexial Dispersion Plots](http://www.nltk.org/book/ch01.html#fig-inaugural) -- and honestly, I'm most excited about this idea. I could see a workflow that uses these visualizers as follows:

- Bar chart - determine if a feature should be eliminated because so many outliers 
- Dispersion plot - determine if instances should be removed or if there is an ordering effect in missing values 

Not sure what the heatmap vision is - but very cool!